tests/ImageSharp.Tests/PixelFormats/La16Tests.cs

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Numerics;
+
+using SixLabors.ImageSharp.PixelFormats;
+using Xunit;
+
+// ReSharper disable InconsistentNaming
+namespace SixLabors.ImageSharp.Tests.PixelFormats
+{
+    public class La16Tests
```

Comment:
I wonder if it's worth to add an integration test doing some arbitrary stuff with `Image<La16>` and `Image<La32>` at least for debug support, or do we just trust the unit tests and the implicit coverage given by code in `PngScanlineProcessor`?

Reply:
I'm gonna leave it for now. The coverage is good via the `PixelOperations` overloads + tests.